pkg/networkservice/core/trace/client.go

```diff
@@ -45,48 +47,47 @@ func (t *traceClient) Request(ctx context.Context, request *networkservice.Netwo
 	ctx = withTrace(ctx)
```

Comment:
Trace chain element should be aware of logger.IsTracingEnabled() and in future it could take value from metadata incoming context to turn off/on logging per request.